resources/definitions/flsun_SuperRacer.def.json

```diff
@@ -1,18 +1,19 @@
 {
     "version": 2,
-    "name": "FlSun SuperRacer",
+    "name": "Flsun Super Racer",
     "inherits": "fdmprinter",
     "metadata": {
         "visible": true,
-        "author": "Thushan Fernando",
+        "author": "Thushan Fernando, Guislain Cyril",
         "manufacturer": "Flsun",
-        "platform": "flsun_sr.3mf",
+        "platform": "flsun_SuperRacer.stl",
+        "platform_offset": [0,-35,-40],
         "file_formats": "text/x-gcode",
         "has_materials": true,
-        "has_machine_quality": false,
+        "has_machine_quality": true,
         "preferred_quality_type": "fast",
         "machine_extruder_trains": {
-            "0": "flsun_sr_extruder_0"
+            "0": "flsun_SuperRacer_extruder_0"
```

Comment:
If you rename this, you will also need to create an upgrade script.

Since this requires a fair amount of python coding, i would not readily recommend doing that.

Reply:
Why ? The release containing the profile has not yet been published, I tested this configuration before creating this PR and it works perfectly.

Reply:
My bad! I didn't check if they were already published. You're completely right.